python/tvm/topi/arm_cpu/conv2d_gemm.py

```diff
@@ -90,7 +91,7 @@ def compute_conv2d_gemm_without_weight_transform(
 
     OH = (IH + pad_top + pad_down - dilated_kernel_h) // HSTR + 1
     OW = (IW + pad_left + pad_right - dilated_kernel_w) // WSTR + 1
-    if pad_top or pad_left:
+    if pad_top or pad_left or pad_down or pad_right:
```

Comment:
This test case was failing before, since it was padding bottom and right, while the schedule would only check top and left: https://github.com/apache/tvm/blob/5e32bf0adf4b8f242bf32a3d3da598cf3083b1ac/tests/python/topi/python/test_topi_conv2d_nhwc.py#L71